oxide-vpc/src/engine/mod.rs

```diff
@@ -212,9 +217,24 @@ impl NetworkParser for VpcParser {
         meta.inner.ip = Some(ip_hi.meta);
         offsets.inner.ip = Some(ip_hi.offset);
 
+        // Allocate a message block and copy in the squashed data. Provide
+        // enough extra space for genve encapsulation to not require an extra
+        // allocation later on. 128 is based on
+        // - 18 byte ethernet header (vlan space)
+        // - 40 byte ipv6 header
+        // - 8 byte udp header
+        // - 8 byte geneve header
+        // - space for geneve options
+        const EXTRA_SPACE: Option<usize> = Some(128);
```

Comment:
Move a bit up and use in ARP case above too

Reply:
Done